apps/backend/src/authn/okta.strategy.ts

```diff
@@ -35,24 +40,26 @@ export class OktaStrategy extends PassportStrategy(Strategy, 'okta') {
         }/oauth2/default/v1/userinfo`,
         clientID: configService.get('OKTA_CLIENTID') || 'disabled',
         clientSecret: configService.get('OKTA_CLIENTSECRET') || 'disabled',
-        callbackURL:
-          `${configService.get('EXTERNAL_URL')}/authn/okta/callback` ||
-          'disabled',
-        scope: 'openid email profile',
-        passReqToCallback: true
+        callbackURL: `${configService.get('EXTERNAL_URL')}/authn/okta/callback`,
+        scope: 'openid email profile'
       },
-      (
-        _req: Request,
-        _token: string,
-        _tokenSecret: string,
-        profile: OktaProfile,
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      async function (
+        // Like in oidc.strategy.ts, we changed the arity of the function to 9 to access the data we need from 'uiProfile' due to updates in the passport-openidconnect library which otherwise caused failures in the authentication process
```

Comment:
Can we please state the meaning of the underscore (_) variables, thanks.

Reply:
Good call! Added it.